internal/cmd/cli/create/computeinstance/create_compute_instance_cmd.go

```
@@ -766,7 +766,7 @@ func (c *runnerContext) applyNetworkingFlags(spec *publicv1.ComputeInstanceSpec_
 		return nil
 	}
 	if c.args.subnet != "" {
-		spec.Subnet = proto.String(c.args.subnet)
+		spec.Subnet = new(c.args.subnet)
```

golangci-lint (2.12.2)

[error] SA1019: spec.Subnet is deprecated: use network_attachments. Single subnet for the VM's primary NIC.

(staticcheck)